### PR TITLE
Checkbox component

### PR DESCRIPTION
**Pull request by kterekh**
Please review.

### CODE REVIEW

File: src/common/Checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="form-check">
+    <input class="form-check-input" type="checkbox" value="" id="flexCheckChecked"
```
Comment (silchencko) [inline src/common/Checkbox.vue:3]: be careful with html `id`, you may use this component many times

File: src/common/Input.vue [hunk 2]
```diff
@@ -26,9 +25,7 @@ import { inputType } from '@/common/types'
 
 @Component
 export default class Input extends Vue {
-  value = ''
   @Prop() type!: inputType
-  @Prop({ default: 'false' }) isDisabled!: boolean
```
Comment (silchencko) [inline src/common/Input.vue:31]: Do you think this input can't be disabled?
maybe `:disabled="isDisabled"`?

File: src/common/Checkbox.vue (hunk 1 above)
Comment (kterekh) [reply src/common/Checkbox.vue:3]: Thanks! Added id as a property, further will pay more attention to this attribute.

File: src/common/Input.vue (hunk 2 above)
Comment (kterekh) [reply src/common/Input.vue:31]: For some reason, I've deleted this property earlier (thinking smaylyk).
Added it again, thanks!

Comment (kterekh) [reply src/common/Input.vue:31]: For some reason, I've deleted this property earlier (thinking emoji).
Added it again, thanks!

Comment (kterekh) [reply src/common/Input.vue:31]: For some reason, I've deleted this property earlier (thinking emoji).
Added it again, thanks!

Comment (kterekh) [reply src/common/Input.vue:31]: For some reason, I've deleted this property earlier (thinking emoji).
Added it again, thanks!

Comment (kterekh) [reply src/common/Input.vue:31]: For some reason, I've deleted this property earlier (thinking emoji). Added it again, thanks!

**Pull request merged**